src/components/Common/Buttons/SeoToggle.tsx

```diff
@@ -0,0 +1,23 @@
+import { FaInfo } from "react-icons/fa";
+import React, { useState } from 'react';
+
+export const SeoToggle = ({ onClick }: { onClick }) => {
+  const [isBgTransparent, setIsBgTransparent] = useState(false);
+
+  const handleClick = () => {
+    onClick(); 
+    setIsBgTransparent(!isBgTransparent); 
+  };
+
+  return(
+<div className="pt-5 pb-5" >
+        <button
+          onClick={handleClick}
+          type="button"
+          className={"seoToggle flex items-center rounded-md bg-violet-700 ml-2 mb-2 px-4 py-2 font-medium text-white shadow-sm focus:outline-none dark:bg-violet-900/[0.6]  dark:text-violet-200"}
+        >
+         Back to Astrostation
```

Comment:
Maybe change to `Back to Astrostation (hide info)`? Not sure I love that though.

Reply:
Options:

![image](https://user-images.githubusercontent.com/93755103/227051647-2c398a8d-3a7f-47cd-9a19-7a417eb169a5.png)
![image](https://user-images.githubusercontent.com/93755103/227051797-3d5b2195-4f66-450f-a309-5c61d4a06ad5.png)
![image](https://user-images.githubusercontent.com/93755103/227051896-0cb76b7a-853f-49f8-8ece-615c1355b3c6.png)
![image](https://user-images.githubusercontent.com/93755103/227051968-19dda6b6-aaaa-47bf-912f-c47134396fac.png)
![image](https://user-images.githubusercontent.com/93755103/227052151-3dbfa736-c725-4044-bb59-b9152ccb67a0.png)
![image](https://user-images.githubusercontent.com/93755103/227052351-a72b4175-7b0b-4b2f-a018-63702ad0bdda.png)
![image](https://user-images.githubusercontent.com/93755103/227052443-0f50a64a-9554-413e-ac77-b30c1e37ecde.png)
![image](https://user-images.githubusercontent.com/93755103/227052512-95a6209c-335c-4fda-820c-e069c2a39b99.png)

Reply:
Any of the options that have some form of hide info/hide section look fine to me. I just felt being a little explicit was a good thing.